.evg.yml

```diff
@@ -74,6 +74,41 @@ buildvariants:
 tasks:
   - name: "build"
     commands:
+      # fetch mongosqltranslate libraries for each platform before building
+      # linux arm
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "arm"
+          platform: "linux"
+          lib_prefix: "lib"
+          ext: "so"
+          release_version: "snapshot"
+      # linux x86
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "x86_64"
+          platform: "linux"
+          lib_prefix: "lib"
+          ext: "so"
+          release_version: "snapshot"
+      # macos arm
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "arm"
+          platform: "macos"
+          lib_prefix: "lib"
+          ext: "dylib"
+          release_version: "snapshot"
+      # macos arm
+      - func: "fetch libmongosqltranslate"
+        vars:
+          arch: "x86_64"
+          platform: "macos"
+          lib_prefix: "lib"
+          ext: "dylib"
+          release_version: "snapshot"
+      # windows
+      - func: "fetch mongosqltranslate windows"
```

Comment:
I have an open thread on the mongosql-rs PR related to these builds suggesting that the windows library follow the same pattern as the other libraries. This would allow us to reuse the same function here as well.

Reply:
I agree. I also chimed in on your comment in the mongosql-rs PR.